core/domain/topic_services_test.py

```diff
@@ -67,6 +68,105 @@ def setUp(self):
         self.user_b = user_services.UserActionsInfo(self.user_id_b)
         self.user_admin = user_services.UserActionsInfo(self.user_id_admin)
 
+    def test_assign_roles_with_invalid_new_role(self):
+        with self.assertRaisesRegexp(Exception, 'Invalid role: %s' % None):
+            topic_services.assign_role(
+                self.user_admin, self.user_a, None, self.TOPIC_ID)
+
+    def test_get_all_topic_rights(self):
+        topic_rights = topic_services.get_all_topic_rights()
+
+        self.assertTrue(topic_rights)
+
+    def test_apply_change_list_with_update_additional_story_ids(self):
+        story_ids = [self.story_id_1, self.story_id_2, self.story_id_3]
+        change_list = [topic_domain.TopicChange({
+            'cmd': topic_domain.CMD_UPDATE_TOPIC_PROPERTY,
+            'property_name': topic_domain.TOPIC_PROPERTY_ADDITIONAL_STORY_IDS,
+            'old_value': None,
+            'new_value': story_ids,
+        })]
+        topic = topic_services.apply_change_list(self.TOPIC_ID, change_list)[0]
+
+        self.assertTrue(topic.additional_story_ids == story_ids)
```

Comment:
Have a check here to assert that other properties for the topic are not changed.

Instead of `assertTrue` user `assertEqual`